pages/article/ArticleHandler.inc.php

```diff
@@ -224,8 +285,12 @@ function downloadSuppFile($args, $request) {
 	 */
 	function download($args, $request) {
 		$articleId = isset($args[0]) ? $args[0] : 0;
-		$galleyId = isset($args[1]) ? $args[1] : 0;
-		$fileId = isset($args[2]) ? (int) $args[2] : 0;
+		$submissionRevision = isset($args[1]) ? (int) $args[1] : 0;
```

Comment:
Yes, goot that you point it out. I did not see that. I removed the `$submissionRevision` parameter form download, so the old urls are still working. But this way the download of old versions is not working, because the redirect (https://github.com/pkp/ojs/blob/master/pages/article/ArticleHandler.inc.php#L171) calls the initialize function which overwrites the `$submissionRevision` and the `download()` function does only hand out the most recent submission revision. Do you know if the redirect is needed or could I replace it with a normal function call like `$this->download()`?